ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/classload/ClassLoadInvoker.java

```diff
@@ -601,14 +607,56 @@ public List<String> availableImports() {
     public List<String> availableVariables() {
         // Available variables and values as string.
         List<String> varStrings = new ArrayList<>();
+        List<String> variablesDeclarations = new ArrayList<>();
+        List<String> finalVariablesDeclarations = new ArrayList<>();
+        List<String> constVariablesDeclarations = new ArrayList<>();
+        String varString;
+
         for (GlobalVariable entry : globalVars.values()) {
             Object obj = InvokerMemory.recall(contextId, entry.getVariableName().getName());
             String objStr = StringUtils.getExpressionStringValue(obj);
             String value = StringUtils.shortenedString(objStr);
-            String varString = String.format("(%s) %s %s = %s",
-                    entry.getVariableName(), entry.getType(), entry.getVariableName(), value);
-            varStrings.add(varString);
+            if (!entry.getQualifiersAndMetadata().isEmpty()) {
+                varString = String.format("(%s) %s %s %s = %s",
+                        entry.getVariableName(), entry.getQualifiersAndMetadata(), entry.getType(),
+                        entry.getVariableName(), value);
+                finalVariablesDeclarations.add(varString);
+            } else {
+                varString = String.format("(%s) %s %s = %s",
+                        entry.getVariableName(), entry.getType(), entry.getVariableName(), value);
+                variablesDeclarations.add(varString);
+            }
+        }
+
+        for (Map.Entry<Identifier, ModuleMemberDeclarationSnippet> entry : availableModuleDeclarations.entrySet()) {
+            if (entry.getValue().getRootNode().kind() == SyntaxKind.CONST_DECLARATION) {
+                ConstantDeclarationNode constantDeclarationNode =
+                        (ConstantDeclarationNode) entry.getValue().getRootNode();
+                String varName = constantDeclarationNode.variableName().text();
+                String constKeyword = constantDeclarationNode.constKeyword().text();
+                String value = constantDeclarationNode.initializer().toString();
+
+                varString = String.format("(%s) %s %s = %s", varName, constKeyword, varName, value);
+                constVariablesDeclarations.add(varString);
+            }
+
         }
+
+        if (variablesDeclarations.size() > 0) {
+            varStrings.add("Variable declarations");
+            varStrings.addAll(variablesDeclarations);
+        }
+
+        if (finalVariablesDeclarations.size() > 0) {
+            varStrings.add("Final variable declarations");
+            varStrings.addAll(finalVariablesDeclarations);
+        }
+
+        if (constVariablesDeclarations.size() > 0) {
+            varStrings.add("Constant Variable Declarations");
```

Comment:
Shall we maintain the lowercase uppercase consistency when naming?  and remove `variable` from `Constant Variable Declarations` since it's not a variable?

Reply:
Fixed in cd59992ad5c45fb100076e2cd13154f643ae153b